src/applications/disability-benefits/686c-674/components/IntroductionPageHeader.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+
+export const IntroductionPageHeader = () => (
+  <>
+    <FormTitle title="Add or remove dependents from your VA benefits" />
```

Comment:
Decided to break this out to make the introduction page a little more manageable.